src/language-js/parse/babel.js

```diff
@@ -62,6 +61,14 @@ const pipelineOperatorPlugins = [
   ["pipelineOperator", { proposal: "minimal" }],
   ["pipelineOperator", { proposal: "fsharp" }],
 ];
+
+/**
+ * The proposal ImportAssertions is now ImportAttributes.
+ * Until a runtime that supports ImportAssertions comes to EoL, we will support it.
+ * @type {Array<ParserPlugin>}
+ */
+const importAttributesPlugins = ["importAttributes", "importAssertions"];
```

Comment:
You can use `["importAttributes", { deprecatedAssertSyntax: true }]` which supports both syntaxes.